pages/docs/_middleware.ts

```diff
@@ -22,10 +22,18 @@ export function middleware(req: NextRequest, ev: NextFetchEvent) {
 
     for (const key of [path, pathIndexHtml, pathHtml]) {
       if (key in docsRedirects) {
+        const destination = docsRedirects[key]
+        
+        if (destination.startsWith('https://')) {
```

Comment:
do we need to check for `http` also?

Reply:
There are not any `http://` redirects defined at the moment, and I wouldn't expect any new ones to get added going forward. I can add it for completeness though 👍